services/datamanager/builtin/file_deletion.go

```diff
@@ -0,0 +1,111 @@
+package builtin
+
+import (
+	"context"
+	"errors"
+	"io/fs"
+	"math"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"go.viam.com/rdk/logging"
+
+	"github.com/ricochet2200/go-disk-usage/du"
+	"go.viam.com/rdk/services/datamanager/datacapture"
+	"go.viam.com/rdk/services/datamanager/datasync"
+)
+
+// TODO change these values back to what they should be
+const (
+	fileDeletionThreshold    = .2
```

Comment:
Perhaps super really descriptive — `fileSystemThresholdToTriggerDeletion`? or `fsThresholdToTriggerDeletion` for consistency/succinctness?